app/assets/stylesheets/users.scss

```
@@ -0,0 +1,69 @@
+@import "bootstrap-sprockets";
+@import "bootstrap";
+
+/* sidebar */
+
+$state-danger-text: #a94442 !default;
+
+aside {
+    section.user_info {
+        margin-top: 20px;
+    }
+    section {
```

```suggestion
    section.user_info {
        margin-top: 20px;
    }
    
    section {
```
E cũng thêm line trống cho các css bên trong group luôn nhé